addons/stock_account/__init__.py

```diff
@@ -39,3 +39,30 @@ def _configure_journals(env):
             data['account.journal'] = ChartTemplate._get_stock_account_journal(template_code)
         ChartTemplate._load_data(data)
         ChartTemplate._post_load_data(template_code, company, template_data)
+
+    # if we already have quantity on hand for storable product, create an initial valuation layer
+    valued_locations = env["stock.location"].search([]).filtered(lambda loc: loc._should_be_valued())
+    qty_by_product = env["stock.quant"]._read_group(
+        [
+            ("product_id.type", "=", "product"),
+            ("location_id", "in", valued_locations.ids),
+            ("owner_id", "=", False),
+        ],
+        groupby=["company_id", "product_id"],
+        aggregates=["quantity:sum"],
+        having=[("quantity:sum", "!=", 0)],
+    )
+    svl_vals = []
+    for company, product, quantity in qty_by_product:
+        value = quantity * product.standard_price
+        svl_vals.append({
+            "company_id": company.id,
+            "product_id": product.id,
+            "description": "Initial Quantity",
+            "quantity": quantity,
+            "value": value,
+            "remaining_qty": quantity if quantity > 0 else 0,
```

Comment:
If quantity is negative, remaining_qty should be too

```suggestion
            "remaining_qty": quantity,
```